src/Exporter.php

```diff
@@ -47,13 +53,26 @@ public function crawl(bool $crawl): self
         return $this;
     }
 
-    public function paths(array $paths): self
+    public function paths(...$paths): self
     {
+        $paths = is_array($paths[0]) ? $paths[0] : $paths;
+
         $this->paths = array_merge($this->paths, $paths);
 
         return $this;
     }
 
+    public function urls(...$urls): self
+    {
+        $urls = is_array($urls[0]) ? $urls[0] : $urls;
+
+        $this->paths(array_map(function (string $url): string {
+            return Str::replaceFirst($this->urlGenerator->to('/'), '', $url);
+        }, $urls));
```

Comment:
Maybe shorten it
```php
$this->paths(array_map(function (string $url): string {
     return Str::replaceFirst($this->urlGenerator->to('/'), '', $url);
}, Arr::wrap($urls)));
```

Reply:
Why is it `$this->paths(array_map(...))` instead of `$this->paths = array_map(...)` ?

Reply:
These methods aren't assignment methods instead they add values.